### ISSUE TITLE
An auto-deny tool for the Admin Queue

### ISSUE THREAD

**Issue by blastoise186** [Dougley/MBv2#114]
**Is your feature request related to a problem? Please describe.**

Sometimes the Admin Queue gets loaded with reports and dupe requests that seemed valid at the time they were submitted, but the post/comment(s) got deleted from the site before an Admin was able to check in and process the queue. This wastes time and confuses people as to why requests got denied.

**Describe the solution you'd like**

It would be useful for all of us if MegaBot could automatically detect when a request has become invalid because the reported post/comment got deleted, then auto-deny it with a suitable reason and pull it out of the admin queue.

**Additional context**

I noticed that #74 is on the open requests for deny reasons, so I'm tagging it here for reference

**Comment by Dougley** [comment]
To make sure, you're talking about automatically removing posts from the queue if we detect suggestions involved with it are deleted? Your issue is worded very ambiguously.

**Comment by blastoise186** [comment]
Yes, that's right. Sorry if the OP wasn't quite as clear as it could be

**Comment by Kitty-Therapist** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Clean deleted content from admin queue](https://trello.com/c/EPR0ldpT/7-clean-deleted-content-from-admin-queue)
